src/test/java/com/fakng/fakngagrgtr/parser/StripeParserTest.java

```diff
@@ -0,0 +1,79 @@
+package com.fakng.fakngagrgtr.parser;
+
+import com.fakng.fakngagrgtr.parser.stripe.StripeParser;
+import com.fakng.fakngagrgtr.persistent.company.Company;
+import com.fakng.fakngagrgtr.persistent.company.CompanyRepository;
+import com.fakng.fakngagrgtr.persistent.vacancy.Vacancy;
+import com.gargoylesoftware.htmlunit.WebClient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+public class StripeParserTest extends AbstractParserTest {
+    private static final String URL = "Stripe";
+    private static final String STRIPE_VACANCY_URL = "https://stripe.com/title_1/111";
+    private static final String DATA_PATH = "stripe-block-data.html";
+    private static final String STRIPE_DATA_PATH = "stripe-data1.html";
```

Comment:
Can filenames a bit better describe what page they represent? For JSON files suffix "data" makes sense because it is just data
But HTML documents are actually pages. And pages usually represent some logical unit. When I look at these names I can't understand which one represent a list of vacancies and which one is for a specific vacancy